### DISCUSSION TITLE
Is it possible to actually get the raw MDX file?

### DISCUSSION THREAD

**Discussion by icyair100** [rohit-gohri/redocusaurus#309] [Q&A]
I would like to generate the MDX file from the yaml spec, and interact with it directly. Is there some setting I can change to allow this?

**Comment by rohit-gohri** [comment]
I don't think that's possible. Because there is no markdown generated, it's a react component that renders the full spec. There is this different feature of importing partial schemas into MDX that might be useful - https://redocusaurus.vercel.app/docs/guides/schema-imports
